woocommerce/woocommerce-gateway-stripe
# Add the WordPress.com Stripe Connect User Account field to settings

Fixes #312 

#### Changes proposed in this Pull Request:
- Simply adds the field to settings. We cannot use the field in the context of the plugin, but API clients like Calypso will find the field value useful if present.

#### To Test:
- Make sure the setting is NOT displayed at `/wp-admin/admin.php?page=wc-settings&tab=checkout&section=stripe`
- Make sure you see the setting in the response to `/wp-json/wc/v2/payment_gateways/stripe`
- Make sure you can update the setting via the API e.g.

<img width="881" alt="screen shot 2017-08-11 at 10 02 13 am" src="https://user-images.githubusercontent.com/1595739/29223663-344cff82-7e7c-11e7-9ad1-97f8c369bacd.png">


-------------------
